{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/careers/models.py

```diff
@@ -76,6 +120,76 @@ class Career(PageBase):
         blank=True,
     )
 
+    date_posted = models.DateField(
+        auto_now_add=True,
+        blank=True,
+        null=True,
+    )
+
+    # Schema Fields
+    employment_type = models.CharField(
+        choices=[
+            ('FULL_TIME', 'Full time'),
+            ('PART_TIME', 'Part time'),
+            ('CONTRACTOR', 'Contract'),
+            ('TEMPORARY', 'Temporary'),
+            ('INTERN', 'Internship'),
+        ],
+        max_length=16,
+        blank=True,
+        null=True,
+    )
+
+    education_requirements = models.CharField(
+        max_length=512,
+        blank=True,
+        null=True,
+    )
+
+    experience_requirements = models.CharField(
+        max_length=512,
+        blank=True,
+        null=True,
+    )
+
+    qualifications = models.CharField(
+        max_length=512,
+        blank=True,
+        null=True,
+        help_text='Specific qualifications required for this role.'
```

Comment:
I don't think this help text is necessary - it doesn't really say anything the field name does not. IMO, either expand it or remove it.